docs/source/tracking.rst

```diff
@@ -345,11 +345,29 @@ Autologging captures the following information:
 
-**Note**: this feature is experimental - the API and format of the logged data are subject to change.
-Moreover, Spark datasource autologging occurs asynchronously - as such, it's possible (though unlikely)
-to see race conditions when launching short-lived MLflow runs that result in datasource information
-not being logged.
+.. note::
+  - This feature is experimental - the API and format of the logged data are subject to change.
+  - Moreover, Spark datasource autologging occurs asynchronously - as such, it's possible (though unlikely) to see race conditions when launching short-lived MLflow runs that result in datasource information not being logged.
+
+Fastai (experimental)
+---------------------
 
+Call :py:func:`mlflow.fastai.autolog` before your training code to enable automatic logging of metrics and parameters.
+See an example usage with `Fastai <https://github.com/mlflow/mlflow/tree/master/examples/fastai>`_.
+
+Autologging captures the following information:
+
++-----------+------------------------+--------------------------------------------------+---------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Framework | Metrics                | Parameters                                       | Tags          | Artifacts                                                                                                                                                             |
++-----------+------------------------+--------------------------------------------------+---------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| fastai    | user-specified metrics | Logs optimizer data as parameters. For exmaple,  |  --           | Model checkpoints are logged to a ‘models’ directory; `MLflow Model`_ (fastai Learner model) on training end; Model summary text is logged                            |
+|           |                        | ``epochs``, ``lr``, ``opt_func``, etc            |               |                                                                                                                                                                       |
+|           |                        | Logs the parameters of the ``EarlyStopping`` and |               |                                                                                                                                                                       |
+|           |                        | ``OneCycleScheduler`` callbacks                  |               |                                                                                                                                                                       |
```

Comment:
Can we insert links for `EarlyStoppingCallback` and `OneCycleScheduler`?

- https://docs.fast.ai/callbacks.html#EarlyStoppingCallback
- https://docs.fast.ai/callbacks.html#OneCycleScheduler

Reply:
Would it make sense to add links here or in the `[mlflow.fastai.autolog(](https://mlflow.org/docs/latest/python_api/mlflow.fastai.html#mlflow.fastai.autolog))`

Or both?

Reply:
![Screen Shot 2020-07-30 at 8 22 02](https://user-images.githubusercontent.com/17039389/88863514-001e1c00-d23e-11ea-8e2b-cc482e1c2001.png)

Reply:
Yes, let's add link in docstring of `mlflow.fastai.autolog` :)